cypress/e2e/home.cy.ts

```diff
@@ -2,7 +2,42 @@ describe('Visits the home page', () => {
     before(() => {
         cy.visit('/')
     })
-    it('Displays the Hero Section', () => {
-        cy.get('[data-testid="searchbutton"]').should('be.visible')
+
```

Comment:
When I trained folks at [PREVIOUS JOB] in Cypress, divvying up the tests and giving everyone a specific area of focus helped everyone make incremental progress